### PR TITLE
convert msc.c to dmsc.d

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/dmsc.d [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(DMDSRC _dmsc.d)
+ */
+
+module ddmd.dmsc;
```
Comment (UplinkCoder) [inline src/dmsc.d]: Could you add a description what dmsc.d does ß

Comment (andralex) [inline src/dmsc.d]: ok to leave it for a future pass

**Comment by andralex** [comment]
Auto-merge toggled on

**Comment by andralex** [comment]
@WalterBright autotester dun like this

**Comment by andralex** [comment]
Auto-merge toggled off

**Comment by UplinkCoder** [comment]
needs a rebase.

**Comment by WalterBright** [comment]
ldc is giving the following error on the travis-ci build:
```
backend/global.d(70): Error: Global variable type does not match previous declaration with same mangled name: configv
```
I don't understand the error message - it works fine on every other platform. @klickverbot any ideas?

**Comment by UplinkCoder** [comment]
@andralex passes.
ready to merge.

**Comment by andralex** [comment]
Auto-merge toggled on

**Pull request merged**